### ISSUE TITLE
OpenVPN Client fStatus

### ISSUE THREAD

**Issue by marcellomarques1986** [rbicelli/pfsense-zabbix-template#20]
Good evening.
I applied all the scrippts however
OpenVPN Client UDP4 Tunnel Status only collects 0 as a value, even though it has an active tunnel it does not collect the information, I tried to apply a javscript but even so the collection did not work.

Congratulations on collecting users by name and traffic.
![Screenshot - 2020-04-27T223941 106](https://user-images.githubusercontent.com/43152741/80437534-344d0e00-88d8-11ea-86f1-5eee33cc4bfa.png)

**Comment by rbicelli** [comment]
Sorry, I made a mistake with value mappings.
I corrected and added new value mapping for openvpn servers in user/auth mode. Now I think everyhing is correct.
Anyway I think you are not clearly understanding a point: when a OpenVPN service is in user auth mode pfSense can't report the status of the tunnel, because there are multiple tunnels initiated by clients.
TL;DR Update to the latest commit

This is the status report in form of an array of a OpenVPN Server in **user auth** mode:
``
    [0] => Array
        (
            [port] => 1194
            [mode] => server_user
            [name] => OpenVPN Contractors UDP4:1194
            [conns] => Array
            [vpnid] => 1
            [mgmt] => server1 
       )
``

And this is the report of **a p2p/Shared key OpenVPN Server**
``
 [1] => Array
        (
            [port] => 1195
            [mode] => p2p_shared_key
            [name] => OpenVPN xxxx UDP4:1195           
            [vpnid] => 2
            [mgmt] => server2
            [status] => up
            [connect_time] => Sat Apr 18 23:17:12 2020
            [virtual_addr] => x.x.x.x
            [remote_host] => x.x.x.x
            [remote_port] => 53034
            [local_host] => 127.0.0.1
            [local_port] => 1195
            [virtual_addr6] => 
            [bytes_recv] => 164783720
            [bytes_sent] => 55986320
        )
``
As you can see there's no status key in OpenVPN in user auth mode, so i added a "fake" status report of up/listening into value mappings.
